Samples/PublicSamples/RecordingBot/src/RecordingBot.Services/Bot/BotMediaStream.cs

```diff
@@ -32,44 +17,13 @@ namespace RecordingBot.Services.Bot
     /// </summary>
     public class BotMediaStream : ObjectRootDisposable
     {
-        /// <summary>
-        /// The participants
-        /// </summary>
         internal List<IParticipant> participants;
-
-        /// <summary>
-        /// The audio socket
-        /// </summary>
         private readonly IAudioSocket _audioSocket;
-        /// <summary>
-        /// The media stream
-        /// </summary>
         private readonly IMediaStream _mediaStream;
-        /// <summary>
-        /// The event publisher
-        /// </summary>
         private readonly IEventPublisher _eventPublisher;
-
-        /// <summary>
-        /// The call identifier
-        /// </summary>
         private readonly string _callId;
-
-        /// <summary>
-        /// Return the last read 'audio quality of experience data' in a serializable structure
-        /// </summary>
-        /// <value>The audio quality of experience data.</value>
         public SerializableAudioQualityOfExperienceData AudioQualityOfExperienceData { get; private set; }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="BotMediaStream" /> class.
-        /// </summary>
-        /// <param name="mediaSession">he media session.</param>
-        /// <param name="callId">The call identity</param>
-        /// <param name="logger">The logger.</param>
-        /// <param name="eventPublisher">Event Publisher</param>
-        /// <param name="settings">Azure settings</param>
-        /// <exception cref="InvalidOperationException">A mediaSession needs to have at least an audioSocket</exception>
```

Comment:
This exception hint can be useful

Reply:
I disagree. These are generated by GhostDoc.

Just because the docs on the method don't indicate an exception that could be thrown here, doesn't mean it cannot throw it. In essence, you will never get an exhaustive list of exceptions that might be thrown, thus, you can't rely on that info anyways. If you think there is any value in such information, get yourself an extension that looks up the top level throw statements from a method on the fly. That way the information will stay up-to-date.

I don't want to keep these generated docs around.

Reply:
An Invalid Operation is Exception is very broad
i dont know what message it carries
but knowing, that at very least one audioSocket needs to be present and knowing that an InvalidOptrationException could be thrown due to the lack of this Socket could be useful information
Developers can avoid running into this by adding an if condition in their code if they know about this
Otherwise, they might expect an ArgumentNullException instead

Reply:
It still does not make sense to keep these docs around. They add to the noise and will inevitably become out of date. They simply duplicate information found in the body below.